assets/utils.tsx

```diff
@@ -19,6 +19,7 @@ import {IArticle, IUser} from 'interfaces';
 // TODO: Improve how we load Moment locales, based on server config
 import 'moment/locale/fr-ca';
 import 'moment/locale/fi';
+import {type IClientConfig} from 'globals';
```

Comment:
why is this imported?

Reply:
because when I added this `companyReportsData: ICompanyReportsData;` to the `globals.d.ts` file, for some reason TS started complaining with some weird generic errors and I was only able to fix them by using specific imports and changing the declaration of the global Window object.

Reply:
On a second thought, perhaps the benefit is that it provides more clarity. Also, there could be times when you would have the same name for a type more than once across the codebase, so an explicit import would be better than an implicit in those cases. What do you think?